### PR TITLE
fix: ui相关问题修改

### PR DESCRIPTION
**Pull request by kingjinni**
ui相关问题修改

Log: ui相关问题修改
Bug: https://pms.uniontech.com/bug-view-260679.html

### CODE REVIEW

**Comment by hudeng-go** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/dde-printer/pull/69#" title="Author self-approved">kingjinni</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdde-printer).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/dde-printer/blob/master/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by hudeng-go** [comment]
### deepin pr auto review
关键摘要:
- 在`BackgroundWidget`类中，`paintEvent`方法中使用`Qt::NoPen`和`Qt::Brush`直接绘制矩形，可能导致性能问题，建议使用更高效的绘制方法。
- `SwitchWidget`类中的`paintEvent`方法重写可能导致性能问题，建议使用更高效的绘制方法。
- `TroubleShootDialog`类中的`m_helpButton`的`setMinimumWidth`和`setMaximumWidth`方法调用可能导致布局问题，建议在调用`setMinimumWidth`和`setMaximumWidth`之前设置`setContentsMargins`。
- 在`TroubleShootDialog`类中，`QVBoxLayout`的`setContentsMargins`被多次调用，建议在循环外部设置一次。

是否建议立即修改:
是

**Review by winexw — APPROVED** [review]

**Pull request merged**
